visualisation/src/pages/MapPage.vue

```diff
@@ -61,8 +63,9 @@ export default Vue.extend({
       selectionOptions: {
         year: {
           name: "Projected Year",
-          min: 2023,
-          max: 2300
+          min: 2020,
+          max: 2150,
+          step: 10
```

Comment:
When I attempted to enter 2020 for the `project_year` in the backend, it resulted in a ValueError. Please ensure this is thoroughly tested in the front-end. Here are my suggested values for these settings:

`min` should be set to `current year + 1` since this is a projection year, and we are looking ahead.
`max` should be set to `2300`, as this is the highest year in the sea level rise table.
`step` should be set to `1` and can be ignored if `1` is the default value.

Reply:
I'll look into this more but max of 2300 was causing a lot of issues with medium confidence.
I won't have time for developing full validation, but that will be coming in the coming months in the Ōtākaro project.

So I am probably going to keep `max` at `2150` for that reason, but look closer at the other options.

Reply:
This is because `low confidence projections` cover up to and include the year `2300`, while `medium confidence projections` cover up to and include the year `2150`. Therefore, the front-end parameter selections need to be dynamic rather than static, so that valid year selections adjust based on the chosen confidence level. Refer to the image below for better clarity:
![image](https://github.com/user-attachments/assets/913650b0-0447-4279-b18c-55df088946ba)

Reply:
Yes, you're right. I had looked into this and tried to find an easy way out without having to implement such a complex feature but it's clear that it's needed. I won't be publishing this paper today.